scripts/android/manifest_permission_inject.js

```diff
@@ -10,27 +10,30 @@ module.exports = function(ctx) {
   var jsonFile = JSON.parse(fs.readFileSync(path.join(__dirname, 'manifest_permission_inject.json')));
   var injectPermissions = jsonFile.basePermission.concat(jsonFile.addPermissions);
 
-  fs.readFile(path.join(ctx.opts.projectRoot,
-    'platforms/android/AndroidManifest.xml'), 'utf-8', function(err, data) {
-    if (err) {
-      return deferral.reject('Read file operation failed');
-    }
+  var manifestPath = 'platforms/android/AndroidManifest.xml';
 
-    if (data.indexOf(jsonFile.addPermissions[0]) !== -1) {
-      console.log('CordovaBlinkUp Plugin: AndroidManifest already injected');
-      return deferral.resolve();
-    }
-
-    var result = data.replace(jsonFile.basePermission, injectPermissions.join('\n\t\t'));
-
-    fs.writeFile(path.join(ctx.opts.projectRoot,
-      'platforms/android/AndroidManifest.xml'), result, 'utf-8', function(err) {
+  fs.readFile(path.join(ctx.opts.projectRoot, manifestPath),
```

Comment:
other than factoring out the manifest path into a variable to be used in the error below, the rest of the changes in this file is whitespace